client_test.go

```diff
@@ -389,6 +390,15 @@ func Test_Client_Stop(t *testing.T) {
 		}
 	}
 
+	t.Run("not started", func(t *testing.T) {
+		t.Parallel()
+		client := newTestClient(ctx, t, newTestConfig(t, nil))
+
+		err := client.Stop(ctx)
+		require.Error(t, err)
+		assert.Equal(t, "client not started", err.Error())
```

Comment:
See lint error here, but we've tried to standardize around using `require` over `assert` everywhere.